pheps/phep-9999.md

```diff
@@ -0,0 +1,161 @@
+The author disagrees as the innovation that is driving the heliophysics field is not in adding more APIs on how to create time objects.
+The scientists creating new innovative methods or finding major discoveries do not want to have to deal with switching APIs to handle very similar time objects.
+All that does is slow them down.
+
+One of the most controversial events within the Linux ecosystem (in the authors short lifetime) was the adoption of [systemd](https://systemd.io/).
+Many of the arguments around that time where similar, it reduces innovation, its too complex, users have to learn a new system.
+However, the maintainers of each major linux distribution embraced systemd as it provided a solid foundation to reduce their own maintenance burden but also having one system meant that one could find detailed documentation or common problems and how to solve them via an internet search.
+
+The adoption of `astropy` is in the authors opinion, the same.
+It provides a new foundation that not only reduces the amount of code within PyHC that handles the same functionality, it allows users to focus on the parts of their work that matter.
+Not only would adding more features to `astropy` benefit the astronomy community, it would in turn benefit the PyHC ecosystem.
+
+# Specification
+
+A dedicated implementation is most likely out of scope for a PHEP, as each package will require bespoke changes after an review.
+
+However, there will be a few common strategies that can be summarized below:
+
+- If a package is using `datetime`, a near one to one drop-in replacement would be quick and is very feasible without dedicated effort.
+- For libraries that provide datetime as inputs, they need to add explicit `astropy.time.Time` support as inputs like for example searching for data.
+- For libraries that provide a custom time object, the deprecated of such an object and support for  `astropy.time.Time` would on the surface be simplest.
+  Another choice is to rebase the existing time object onto `astropy.time.Time` to avoid breaking existing user code directly.
+
+It is important to note that for new time formats, they do not need to added to `astropy`, they can be in the same PyHC package as they are currently.
+The time framework has been designed to be extendable without adding code changes to `astropy`.
+
+As the merger of this PHEP implies that the PyHC ecosystem are *planning* on standardizing on `astropy.time.Time`, there will be need to be a ecosystem wide plan to enable this transition.
+
+One such transition plan is as follows:
+
+- Review of the existing core PyHC packages to figure out:
+  - What time objects are used?
+  - How would adding `astropy.time.Time` support as function inputs work?
+  - Figure out missing time formats
+  - Figure out missing features from `astropy.time.Time`
+- Figure out how to deal with `astropy.time.Time` lack of support in pandas
```

Comment:
```suggestion
- Figure out how to deal with `astropy.time.Time` lack of support in pandas and polars
```
In case anyone hasn't heard of it, [polars](https://pola.rs/) is similar to pandas, except that it is written in Rust for better performance.